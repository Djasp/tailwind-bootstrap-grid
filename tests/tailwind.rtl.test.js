const run = require('./run-test');

it('should handle rtl', () => run(__filename));
