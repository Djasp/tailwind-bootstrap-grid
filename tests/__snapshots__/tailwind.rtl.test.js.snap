// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle rtl 1`] = `
"
"
`;
